pkg/cli/doctor.go

```diff
@@ -201,6 +201,11 @@ SELECT id, descriptor, NULL AS mod_time_logical
 FROM system.descriptor ORDER BY id`
 		}
 	}
+	if err == nil {
```

Comment:
Here and below you can use `errors.CombineErrors` and simplify this

Reply:
good call. actually, the error checking was wrong before, since it didn't do anything if `err` was non-nil and also not an `UndefinedColumn` error

Reply:
well, actually CombineErrors is tricky since calling `rows.Close()` will throw a NPE if `err` is non-nil. so we need to check if `err` is nil separately anyway.

i changed it a bit to use `QueryRow` instead, which will take care of closing rows if needed.

there was also a bug because the "column exists" check (`SELECT crdb_internal_mvcc_timestamp`) would always fail -- it needs a FROM clause.

Reply:
sgtm